### PR TITLE
Use `requireModule('ember')` instead of `window.Ember`

### PR DESCRIPTION
**Pull request by rwwagner90**

### CODE REVIEW

File: ember_debug/adapters/basic.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* globals requireModule */
 /* eslint no-console: 0 */
-const Ember = window.Ember;
+const Ember = require('ember') || window.Ember;
```
Comment (rwjblue) [inline ember_debug/adapters/basic.js:3]: I think we need something like:

```suggestion
require.has('ember') ? require('ember') : window.Ember;
```

Comment (rwwagner90) [reply ember_debug/adapters/basic.js:3]: Unfortunately this causes errors

**Comment by rwwagner90** [comment]
@pzuraq @rwjblue I had to wrap the requires in `try/catch` to get things working here. I also needed to skip a couple mixin tests. Do we think that is okay for now, since we discourage mixins anyway?

**Review by pzuraq — APPROVED** [review]

File: ember_debug/adapters/basic.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
-/* globals requireModule */
 /* eslint no-console: 0 */
-const Ember = window.Ember;
+let Ember;
```
[the remark this answers is not in the document]
Comment (rwwagner90) [reply ember_debug/adapters/basic.js:2]: We definitely could! @avihai-yosef would you be interested in opening a PR to do that?

[the remark this answers is not in the document]
Comment (avihai-yosef) [reply ember_debug/adapters/basic.js:2]: Yeah, Sure

**Pull request merged**